### PR TITLE
Added CPP code for usaco-1131

### PR DESCRIPTION
**Pull request by RaheezBZ**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/usaco-1131.mdx [hunk 1]
```diff
@@ -38,4 +38,49 @@ print(length)
...
+using namespace std;
+
+int main(){
+	
+	int n, l;
+	cin >> n >> l;
+	
+	vector<int> citations;
+	int x;
+	while(cin >> x){
```
Comment (envyaims) [inline solutions/usaco-1131.mdx:55]: ? they give you $n$ for a reason

**Review by envyaims — CHANGES_REQUESTED** [review]
ill let the others fix the spacings...

**Comment by SansPapyrus683** [comment]
yeah please space out your ops

File: solutions/usaco-1131.mdx [hunk 2]
```diff
@@ -38,4 +38,46 @@ print(length)
...
+	cin >> n >> l;
+	
+	vector<int> citations(n);
+	for (int i = 0; i < n; i++){
+		cin >> citations[i];
+	}
+	sort(citations.begin(), citations.end(), greater<int>());
+	int length = citations.size();
+	
+	while(length >= 0 && citations[length-1] < length) length -= 1;
```
Comment (envyaims) [inline solutions/usaco-1131.mdx:59]:
```suggestion
	while (length >= 0 && citations[length-1] < length) {
		length -= 1;
	}
```

File: solutions/usaco-1131.mdx [hunk 3]
```diff
@@ -38,4 +38,46 @@ print(length)
...
+	 
+	int cur_h = length;
+	length = citations.size();
+	
+	if(cur_h != n){
```
Comment (envyaims) [inline solutions/usaco-1131.mdx:64]:
```suggestion
	if (cur_h != n) {
```

File: solutions/usaco-1131.mdx [hunk 4]
```diff
@@ -38,4 +38,46 @@ print(length)
...
+		for(int i = cur_h; i >= 0; i--){
```
Comment (envyaims) [inline solutions/usaco-1131.mdx:65]:
```suggestion
		for (int i = cur_h; i >= 0; i--) {
```

File: solutions/usaco-1131.mdx [hunk 5]
```diff
@@ -38,4 +38,46 @@ print(length)
...
+			if(i<= cur_h-1){
```
Comment (envyaims) [inline solutions/usaco-1131.mdx:66]:
```suggestion
			if (i <= cur_h-1) {
```

**Review by envyaims — CHANGES_REQUESTED** [review]
do the same for others pls

File: solutions/usaco-1131.mdx [hunk 6]
```diff
@@ -38,4 +38,46 @@ print(length)
...
+				break;
+			}
+			citations[i] += 1;
+		}
+	}
+	sort(citations.begin(), citations.end(), greater<int>());
+	
+	while(length >= 0 && citations[length-1] < length) length -= 1;
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-1131.mdx:74]:
```suggestion
	while (length >= 0 && citations[length - 1] < length) {
		length -= 1;
	}
```

File: solutions/usaco-1131.mdx [hunk 7]
```diff
@@ -38,4 +38,46 @@ print(length)
...
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
+int main(){
+	
+	int n, l;
+	cin >> n >> l;
+	
+	vector<int> citations(n);
+	for (int i = 0; i < n; i++){
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-1131.mdx:53]:
```suggestion
	for (int i = 0; i < n; i++) {
```

File: solutions/usaco-1131.mdx [hunk 8]
```diff
@@ -38,4 +38,46 @@ print(length)
 ```
         
 </PySection>
+
+<CPPSection>
+
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
+int main(){
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-1131.mdx:47]:
```suggestion
int main() {
```

File: solutions/usaco-1131.mdx [hunk 9]
```diff
@@ -38,4 +38,46 @@ print(length)
 ```
         
 </PySection>
+
+<CPPSection>
+
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-1131.mdx:46]:
```suggestion
using namespace std;

```

File: solutions/usaco-1131.mdx (hunk 2 above)
Comment (Aadit-Ambadkar) [reply solutions/usaco-1131.mdx:59]:
```suggestion
	while (length >= 0 && citations[length - 1] < length) {
		length -= 1;
	}
```

**Review by Aadit-Ambadkar — CHANGES_REQUESTED** [review]

**Comment by RaheezBZ** [comment]
please do merge my pull request

File: solutions/usaco-1131.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-1131.mdx]: unresolved

File: solutions/usaco-1131.mdx [hunk 10]
```diff
@@ -38,4 +38,49 @@ print(length)
 ```
         
 </PySection>
+
+<CPPSection>
+
+```cpp
+#include<bits/stdc++.h>
```
Comment (SansPapyrus683) [inline solutions/usaco-1131.mdx:45]:
```suggestion
#include <bits/stdc++.h>
```

Comment (RaheezBZ) [reply solutions/usaco-1131.mdx]: Fixed spaces now please merge my pull request

**Pull request merged**